nixos/lib/eval-config-standard.nix

```
@@ -0,0 +1,68 @@
+# DO NOT IMPORT. Use nixpkgsFlake.lib.nixos, or import (nixpkgs + "/nixos/lib")
+{ lib }: # read -^
+
+let
+
+  # Base: Use the miminal module evaluation
+  evalModulesMinimal = (import ./default.nix {
+    inherit lib;
+    # Implicit use of feature is noted in implementation.
+    featureFlags.minimalModules = { };
+  }).evalModules;
```

We should have something like this:

```suggestion
  evalModulesMinimal = (lib.withFeatureFlags { minimalModules = { }; }).evalModules;
```

---

Wait, `lib` is `nonExtendedLib` do we really want to add feature flagging to that?!?

Currently, that's only used for `nixos` and we probably should make that an independent decision.